### PR TITLE
[fix] fix compile error with clang

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

fix compile error

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/agent/cgroups_mgr.cpp [hunk 1]
```diff
@@ -420,11 +420,13 @@ Status CgroupsMgr::relocate_tasks(const string& src_cgroups, const string& dest_
     string dest_tasks_path = dest_cgroups + "/tasks";
     std::ifstream src_tasks(src_tasks_path.c_str());
     if (!src_tasks) {
-        return Status::InternalError("Src tasks is null");;
+        return Status::InternalError("Src tasks is null");
+        ;
```
Comment (morningman) [inline be/src/agent/cgroups_mgr.cpp:424]: remove

File: be/src/agent/cgroups_mgr.cpp [hunk 2]
```diff
@@ -420,11 +420,13 @@ Status CgroupsMgr::relocate_tasks(const string& src_cgroups, const string& dest_
...
     }
     std::ofstream dest_tasks(dest_tasks_path.c_str(), std::ios::out | std::ios::app);
     if (!dest_tasks) {
-        return Status::InternalError("Desk task is null");;
+        return Status::InternalError("Desk task is null");
+        ;
```
Comment (morningman) [inline be/src/agent/cgroups_mgr.cpp:429]: remove

**Review by morningman — APPROVED** [review]
LGTM

**Comment by yangzhg** [comment]
merge it to fix master compile error

**Pull request merged**